draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -399,51 +401,53 @@ transports. This document does not change the informational status of those
+It is hoped this situation will change.
 
 To the extent that behavior of widely deployed algorithms is understood, proposals
-can analyze and simulate the their interaction with those algorithms. To the
+can analyze and simulate their interaction with those algorithms. To the
 extent they are not, experiments can be conducted where possible.
 
 Note that in many deployments, real-time traffic is directed into distinct
 queues via Differentiated Services Code Points (DSCP) or other mechanisms,
-which substantially reduces the interplay with other traffic. However, proposals
-cannot assume that this is always the case.
+which substantially reduces the interplay with other traffic. However, a proposal
+targeting Internet use
+MUST NOT assume that all paths support specific mechanisms.
 
 ### Short and Long Flows
 
 The effect on short-lived and long-lived flows using other common congestion
-controls MUST be evaluated, as in {{short-flows}}.
+control algorithms MUST be evaluated, as in {{short-flows}}.
 
 ## Other Criteria
 
 ### Differences with Congestion Control Principles
 
-Proposed congestion control mechanisms SHOULD include a clear
-explanation of the deviations from {{!RFC2914}} and {{!RFC7141}}.
+Proposed congestion control algorithms SHOULD include a clear
+explanation of any deviations from {{!RFC2914}} and {{!RFC7141}}.
 
-### Incremental Deployment.
+### Incremental Deployment
 
-The proposal should discuss whether the alternate congestion
-control mechanism allows for incremental deployment in the
+The proposal ought to discuss whether the proposal
+allows for incremental deployment in the
 targeted environment.  For a mechanism targeted for deployment in
-the current Internet, it would be helpful for the proposal to
+the current Internet, it would be helpful for a proposal to
 discuss what is known (if anything) about the correct operation
-of the mechanism with some of the equipment installed in the
+of the mechanisms with some of the equipment that can be installed in the
 current Internet, e.g., routers, transparent proxies, WAN
 optimizers, intrusion detection systems, home routers, and the
 like.
 
-As a similar concern, if the alternate congestion control
-mechanism is intended only for specific environments (and not the
+As a similar concern, if the proposal
+is intended only for specific environments (and not the
 global Internet), the proposal should consider how this intention
-is to be carried out.  The community will have to address the
-question of whether the scope can be enforced by simply stating
-the restrictions or whether additional protocol mechanisms are
-required to enforce the scoping.  The answer will necessarily
-depend on the change being proposed.
+is to be realised.  The community will have to address the
+question of whether the scope can be enforced by stating
+a ste of restrictions or whether additional protocol mechanisms are
```

Comment:
What does "a ste of" mean? Could this be a typo?

Reply:
whoops